packaging/deb-systemd/debian/changelog

```diff
@@ -0,0 +1,5 @@
+mackerel-agent (0.41.1-1.systemd) stable; urgency=low
+
+   * dummy release to prepare systemd support.
```

Comment:
Simplified at https://github.com/mackerelio/mackerel-agent/pull/360/commits/e8a5dc33814d0e0e78a9b93a6f64ddc86f7348a0 (also bumped this **dummy** version to 0.41.2).